shopfloor_base/models/shopfloor_app.py

```diff
@@ -278,7 +278,7 @@ def _prepare_endpoint_vals(self, service, method_name, route, routing_params):
         return endpoint_vals
 
     def _route_group(self):
-        return f"{self._name}:{self.tech_name}"
+        return f"{self._name}:{self.tech_name}"  # noqa
```

Comment:
Why `noqa`? String seems short enough

Reply:
The stupid linter thinks that you need a space after `:` :rofl: